app/src/main/java/de/tum/in/tumcampusapp/component/other/generic/ImageViewTouchFragment.kt

```diff
@@ -0,0 +1,57 @@
+package de.tum.`in`.tumcampusapp.component.other.generic
+
+import android.graphics.Color
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.Fragment
+import com.squareup.picasso.Callback
+import com.squareup.picasso.Picasso
+import de.tum.`in`.tumcampusapp.R
+import de.tum.`in`.tumcampusapp.utils.Utils
+
+class ImageViewTouchFragment : Fragment() {
+
+    lateinit var mRootView: View
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        mRootView = inflater.inflate(R.layout.fragment_image_view_touch, container, false)
```

Comment:
You can get rid of `mRootView` and then use Kotlin's synthetic properties when loading the image.